src/sdk/model/vm/vm-copy-move-request.ts

```diff
@@ -0,0 +1,78 @@
+import { VmCopyMoveRequestJson, VmCopyMoveVmVnicRequestJson } from './__json__';
+import { VmCopyMoveVmVnicRequest } from './vm-copy-move-vm-vnic-request';
+
+/* istanbul ignore next: autogenerated */
+export class VmCopyMoveRequest {
+
+  private readonly _json: VmCopyMoveRequestJson;
```

Comment:
Why read only for _json ? I guess it could be possible to only refresh the object so we would be able to update this prop.

Reply:
it is never updated so it is correct to be marked read only